.typos.toml

```
@@ -6,4 +6,5 @@
 "O_WRONLY" = "O_WRONLY"
 # From faq.rst, in "mount.s3ql_crit.log"
 "crit" = "crit"
+"*e*nd" = "*e*nd"
```

```suggestion
[default]
extend-ignore-re = [
    '"\*e\*nd"'
]
```

I think this regexp may ignore that "end".

---

Rephrased the comment instead.